ui/pages/map.tsx

```diff
@@ -0,0 +1,287 @@
+import { CITIZEN_ADDRESSES, DEFAULT_CHAIN } from 'const/config'
+import { blockedCitizens } from 'const/whitelist'
+import dynamic from 'next/dynamic'
+import { useRouter } from 'next/router'
+import { useState } from 'react'
+import { generatePrettyLinks } from '@/lib/subscription/pretty-links'
+import { initSDK } from '@/lib/thirdweb/thirdweb'
+import { getAttribute } from '@/lib/utils/nft'
+import IconOrg from '@/components/assets/IconOrg'
+import Container from '@/components/layout/Container'
+import ContentLayout from '@/components/layout/ContentLayout'
+import Frame from '@/components/layout/Frame'
+import Head from '@/components/layout/Head'
+import { NoticeFooter } from '@/components/layout/NoticeFooter'
+import StandardButton from '@/components/layout/StandardButton'
+import Tab from '@/components/layout/Tab'
+
+const Earth = dynamic(() => import('@/components/globe/Earth'), { ssr: false })
+const Moon = dynamic(() => import('@/components/globe/Moon'), { ssr: false })
+
+export default function NetworkMap({
+  citizensLocationData,
+}: {
+  citizensLocationData: any
+}) {
+  const router = useRouter()
+
+  const [tab, setTab] = useState('earth')
+
+  const descriptionSection = (
+    <div className="pt-2">
+      <div className="mb-4">
+        Explore the network map and discover moon landings!
+      </div>
+      <div className="flex gap-4">
+        <Frame className="w-[300px]" noPadding>
+          <div className="flex flex-wrap text-sm bg-filter">
+            <Tab tab="earth" setTab={setTab} currentTab={tab}>
```

Comment:
Should we default to the network view? This is a very cool feature, I'd like to surface it easily in the app.

Not sure, maybe Ryan has some thoughts about how to structure this at the forefront v.s. a click that people might miss if they don't poke around.